### DISCUSSION TITLE
Following modules were not evaluated; make sure they are imported from other code

### DISCUSSION THREAD

**Discussion by Cassielxd** [denoland/deno#26077] [Q&A] [answered]
Version: Deno x.x.x
Custom ext extension Tips Following modules were not evaluated; make sure they are imported from other code
![image](https://github.com/user-attachments/assets/8949e96e-3e2a-4102-978e-99fe7babe37f)
![image](https://github.com/user-attachments/assets/cd94642b-1fe0-4019-b45a-043c5efcd370)
![image](https://github.com/user-attachments/assets/fdb1eb63-762a-4439-998a-04e1dc742491)

**Comment by bartlomieju** [comment, marked as answer]
You need to import `ext:deno_ipc/01_ipcs.js` to one of the files that are evaluated when the runtime is bootstrap - eg. one of the files that are specified in `esm_entry_point` option.

**Reply by Cassielxd** [reply]
The startup file is in deno_runtime's 99_main.js, I don't know how to add it, is there a good way
